docs/installation/bootstrap.rst

```diff
@@ -80,6 +81,15 @@ notation for it's various subfields.
         network. This is an :ref:`advanced configuration<multiple CIDR network>`
         which we do not recommend for non-experts.
 
+      For ``controlPlane`` entry, an ``ingressIP`` can also be provided. This
+      IP is used to connect to all the control plane components, like
+      MetalK8s-UI and the whole monitoring stack. We suggest using
+      a `Virtual IP <https://en.wikipedia.org/wiki/Virtual_IP_address>`_ that
+      will sit on a working master Node. The default value for this
+      Ingress IP is the control plane IP of the Bootstrap node (which means
+      that if you lose the Bootstrap node, you no longer have access to any
+      control plane component).
```

Comment:
...through the Ingress controller. But some components also listen on host ports directly, no? Especially K8s API, which is useful to access in case of Bootstrap issues.

Reply:
Right, but if you lose bootstrap you do not have access to OIDC so access to Kubernetes APIServer (should) do not work as well (if we generated kubeconfig specific to each user :wink:)
But agree, I will replace "control plane component" with `OIDC and various Control Plane UIs`